pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
-name = "robusta-cli"
-version = "0.0.0"
+name = "robusta-api"
+version = "0.0.3"
```

Comment:
In the example in `playbooks-repositories.rst` the version is `1.0`
what is the correct one?

Reply:
It's a mistake, it should stay 0.0.0 in the pyproject.toml file and in the example it should use ^0.14. I've fixed it.